lib/matplotlib/ticker.py

```diff
@@ -475,6 +475,24 @@ def __init__(self, useOffset=None, useMathText=None, useLocale=None):
         self._usetex = mpl.rcParams['text.usetex']
         if useMathText is None:
             useMathText = mpl.rcParams['axes.formatter.use_mathtext']
+            if useMathText is False:
+                try:
+                    ufont = mpl.font_manager.findfont(
+                        mpl.font_manager.FontProperties(
+                            mpl.rcParams["font.family"]
+                        ),
+                        fallback_to_default=False,
+                    )
+                except ValueError:
+                    ufont = None
+
+                if ufont is not None and ufont == str(
```

Comment:
you don't need the None check here

Reply:
I was under the impression that we shouldn't compare values if one of them is None...
I'll update 👍🏼